artist-service/src/main/java/servlet/ArtistServlet.java

```diff
@@ -155,10 +155,22 @@ public void doPut(HttpServletRequest request, HttpServletResponse response) thro
             nickname = parts1[1];
             String[] parts2 = args[1].split("="); //firstname = parts2[1]
             firstName = parts2[1];
-            String[] parts3 = args[2].split("="); //lasttname = parts3[1]
+            String[] parts3 = args[2].split("="); //lastname = parts3[1]
```

Comment:
Can you change the naming, I think parts1, 2, 3, 4 is very confusing. I know you have your comments but still we should not name variables like this....

Reply:
Done!